src/pages/AdminPage/components.tsx

```diff
@@ -291,19 +295,64 @@ export const TableLink = styled(RouterLink, {
   textDecoration: 'none',
 });
 
+const coordinapeTooltipContent = () => {
+  return (
+    <Box
+      css={{
+        m: '$sm',
+        display: 'flex',
+        flexDirection: 'column',
+      }}
+    >
+      <Text size="large" css={{ my: '$md' }}>
+        Why is Coordinape in your circle?
+      </Text>
+      <p>
+        We&apos;re experimenting with the gift circle mechanism as our revenue
+        model. By default, Coordinape appears in your circle and any user can
+        allocate to Coordinape. To remove the Coordinape user, click the trash
```

Comment:
Change this from "To remove..." to "To show or hide the Coordinape user, use the links on the right side of this row."